src/pages/components/notification/code.mdx

```diff
@@ -4,66 +4,142 @@ description: Notifications are messages that communicate information to the user
+
+function MyComponent() {
+  return (
     <InlineNotification
       kind="info"
-      actions={<NotificationActionButton>Action</NotificationActionButton>}
       iconDescription="describes the close button"
-      subtitle={<span>Subtitle text goes here. <a href="#example">Example link</a></span>}
+      subtitle={
+        <span>
+          Subtitle text goes here. <a href="#example">Example link</a>
+        </span>
+      }
       title="Notification title"
     />
-  </div>
-`}</ComponentVariant>
-</ComponentDemo>
+  );
+}
+```
+
+## Component API
+
+### Inline Notification props
+
+| Prop                  | Type       | Required | Default               | Description                                                            |
+| --------------------- | ---------- | -------- | --------------------- | ---------------------------------------------------------------------- |
+| actions               | `node`     | -        | -                     | Pass in the action nodes that will be rendered inside the notification |
+| children              | `node`     | -        | -                     | Pass in the children that will be rendered inside the notification     |
+| className             | `string`   | `true`   | `error`               | Optional class names to be applied to the notification                 |
+| kind                  | `string`   | `true`   | `error`               | Specify what state the notification represents                         |
+| lowContrast           | `bool`     | -        | –                     | Specify whether you are using the low contrast variant styles          |
+| title                 | `string`   | `true`   | `provide a caption`   | Specify notification title                                             |
+| subtitle              | `node`     | –        |                       | Specify notification subtitle                                          |
+| role                  | `string`   | –        | `alert`               | Provide an alternative role for the notification                       |
+| onCloseButtonClick    | `function` | –        | `() => {}`            | Function that is called when notification closes                       |
+| iconDescription       | `string`   | `true`   | `closes notification` | Provide screen reader accessible description for close icon            |
+| statusIconDescription | `string`   | –        | –                     | Provide screen reader accessible description for status icon           |
+| notificationType      | `string`   | –        | `inline`              | Specify the type of the button                                         |
+| hideCloseButton       | `bool`     | –        | `false`               | Optional prop to disable the close button                              |
+
+## Inline notifications with action button
+
+An inline notification optionally can have a ghost button that allows the user to take further action.
+
+```jsx
```

Comment:
Seems like this snippet got all lower-cased (if you were using vim, I totally have had this happened before lol)